pandas/tests/frame/methods/test_drop.py

```diff
@@ -107,6 +108,7 @@ def test_drop_names(self):
         expected = Index(["a", "b", "c"], name="first")
         tm.assert_index_equal(dropped.index, expected)
 
+    @pytest.mark.xfail(is_numpy_dev, reason="GH#39089 Numpy changed dtype inference")
```

Comment:
can u add an issue number to all of these (open one);

we might need to backport this as well

also see if u can find a linked numpy issue - this is a breaking change and might want to have them revert this

Reply:
#39089 is the open issue, you meaning this?

Yeah backport is probably useful. Linked a numpy pr in #39089